README.md

```diff
@@ -1,47 +1,75 @@
-# conu
-__CON__ -tainers __U__ -tils
-
-It provides low level __API__ for docker image/container testing.
-It is based on bash commands because primary target is to __test it
-as customers will__ use it.
-That's why it does not use python docker bindings.
-It provides python classes for:
-- __docker__ module
-  - __Image__ - image handling
-  - __Container__ - docker container handling
-- __utils__ module
-  - __Volume__ - it is mktemd utility on steroids
-  - __Probe__ - provides, simple port/file checks and waiting for this events
+# CONU
+
+Conu (CONtainer Utils) provides low level __API__ for image/container testing.
+TODO - find better punchline :)
 
 # Installation
 
-## Prerequisites
-- system packages providing binaries: `docker`, `setfacl`, `chmod`, `chcon`
-```
-$ dnf -y install acl coreutils docker
-```
-- python modules inside: No specific requirements now
+You can try out CONU in your local machine by running:
 
-## From source Code
-- Commands in git directory:
-```
-$ sudo make install
+```commandline
+git clone https://github.com/fedora-modularity/conu
+sh -c conu/requirements.sh
```

Comment:
damn, this will only work on fedora :/

Reply:
What platform are we planning to use? CentOS, RHEL, Fedora? For the first one we can provide some docu.

Reply:
s/use/support/

I would say all ours (Fedora, CentOS, EL) and other platforms are best effort (Debian-based, Arch, Gentoo, Suse, BSD, Windows, MAC, dos, amiga).

The truth is that once we package this in Fedora, we will rewrite that script so we can utilize RPM deps.

Reply:
I also trough about that. there is not easy way how to make it universal and running on debian based distros. because of different packages, missing packages, various tooling around packaging, it is same also for container-images.
So I thing that solution is, that in case you expect to be running on another distros, make requirements.sh aware of that and check and handle env.